third-party/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ option(USE_BUNDLED_MSGPACK "Use the bundled msgpack." ${USE_BUNDLED})
 option(USE_BUNDLED_LUAJIT "Use the bundled version of luajit." ${USE_BUNDLED})
 option(USE_BUNDLED_LUAROCKS "Use the bundled version of luarocks." ${USE_BUNDLED})
 
+option(ALLOW_EXISTING_SRC_DIR "Skip download of deps sources in case of existing source directory." OFF)
```

Comment:
Not usually, no, but here I thought it would, as e.g. `USE_BUNDLED_LUAROCKS` "forces" the use of the bundled version, i.e. if something goes wrong it doesn't fall back to system-installed one.

Anyway, changed that.